samples/LoggingTracer/LoggingTracer/LoggingTracer.cs

```diff
@@ -23,16 +27,125 @@ internal LoggingTracer()
 
         public ITextFormat TextFormat => new LoggingTextFormat();
 
-        public ISpanBuilder SpanBuilder(string spanName)
-        {
-            Logger.Log($"{prefix}.SpanBuilder({spanName})");
-            return new LoggingSpanBuilder(spanName, SpanKind.Internal);
-        }
-
         public IDisposable WithSpan(ISpan span)
         {
             Logger.Log($"{prefix}.WithSpan");
             return new CurrentSpanUtils.LoggingScope(span);
         }
+
+        public ISpan StartRootSpan(string operationName)
```

Comment:
now all links needs to be set in "builder". So we will need more overloads

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#add-links

Reply:
you can use overloads that dop not have links - you don't have to set them if you don't have them.

we have all methods having overload with links. 

We'll also need to do lazy links - this is in TODO